docs/source/tutorials/low-latency.rst

```diff
@@ -2,7 +2,8 @@ Low Latency
 ===========
 
 These tables show what performance you can expect from **USB 3.2** Gen 1 (5 Gbps) connection with an OAK camera. XLink chunking was
-disabled for these tests (:code:`pipeline.setXLinkChunkSize(0)`).
+disabled for these tests (:code:`pipeline.setXLinkChunkSize(0)`). Example code for testing low-latency of color stream at 1080P,
+MJPEG can `be found here <https://gist.github.com/Erol444/0210c8af650ce8d15276729dcc21ce81>`__.
```

Comment:
Thoughts about adding to examples?

Reply:
Not sure if immediately relevant, as the only unique thing in code (compared to other examples) is setting the xlink chunk size. Thoughts @themarpe ?

Reply:
What about the actual measurement of latency, e.g. how to measure it?

Reply:
@szabi-luxonis IMO it's sufficiently mentioned here:
![image](https://user-images.githubusercontent.com/18037362/181478138-56398968-80df-42a0-9c4f-15176401445e.png)

Reply:
We have a bunch of nice examples, that are unofficial, on gist.
I think adding a simple example of how to measure latency, would be great.
Not many people get very deep into docs, just check what examples are available, so they don't have to bother with the API to measure latency and so on, just run it, or build on top of an example, which is known to be good and supported.